mtp_noms_ops/apps/settings/views.py

```diff
@@ -152,3 +152,34 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['prisons'] = self.request.user_prisons
         return context
+
+
+class JobInformationView(SuccessURLAllowedHostsMixin, FormView):
+    title = _('Help us improve this service')
+    template_name = 'settings/job-information.html'
+    form_class = JobInformationForm
+
+    def get_success_url(self):
+        if REDIRECT_FIELD_NAME in self.request.GET:
+            next_page = self.request.GET[REDIRECT_FIELD_NAME]
+            url_is_safe = is_safe_url(
+                url=next_page,
+                allowed_hosts=self.get_success_url_allowed_hosts(),
+                require_https=self.request.is_secure(),
```

Comment:
this looks like a sensible solution. i think in other places we've used `is_safe_url` https is always required which means it's impossible to test and demo locally

Reply:
I assume we've got some SSL enforcement in the cloud platform set up though so this would report correctly as enforced SSL in production?

Reply:
yeah you cannot access stuff in cloud platform without using ssl